torch/distributed/distributed_c10d.py

```diff
@@ -230,20 +230,28 @@ def _check_tensor_list(param, param_name):
 
 def is_mpi_available():
     """
-    Checks if MPI is available
+    Checks if the MPI backend is available.
 
     """
     return _MPI_AVAILABLE
 
 
 def is_nccl_available():
     """
-    Checks if NCCL is available
+    Checks if the NCCL backend is available.
 
     """
     return _NCCL_AVAILABLE
 
 
+def is_gloo_available():
+    """
+    Checks if the Gloo backend is available.
+
+    """
+    return True
```

Comment:
This seems wrong? IIRC you can build PyTorch without gloo, so we should have a check for that

Reply:
@apaszke You can, but right now it won't build c10d, so it won't load `torch.distributed` at all. This should be fix in and by itself and then this function can be updated as well (added #18851).